src/main/java/com/songoda/epicfarming/utils/DataHelper.java

```diff
@@ -80,15 +80,17 @@ public static List<ItemStack> loadItemsForFarm(int farmId) {
                     .fetchInto(String.class);
 
             for (String serializedItem : serializedItems) {
-                try {
-                    List<ItemStack> deserializedItems = ItemSerializer.fromBase64(serializedItem);
-                    if (deserializedItems != null && !deserializedItems.isEmpty()) {
-                        items.addAll(deserializedItems);
-                    } else {
-                        plugin.getLogger().severe("❌ Failed to deserialize item: " + serializedItem);
+                if (!serializedItem.isEmpty() && serializedItem != null) {
```

Comment:
I think when `serializedItem` is `null` here, the `#isEmpty` call should cause the same problem that you were trying to fix, just somewhere else.

Reply:
good point